Development.md

```diff
@@ -0,0 +1,196 @@
+# Development
```

Comment:
Maybe add a brief summary here about the purpose of this document? Otherwise it's two headers with nothing in between.